src/bitcoin-cli.cpp

```diff
@@ -642,7 +643,8 @@ class NetinfoRequestHandler : public BaseRequestHandler
         "  send     Time since last message sent to the peer, in seconds\n"
         "  recv     Time since last message received from the peer, in seconds\n"
         "  txn      Time since last novel transaction received from the peer and accepted into our mempool, in minutes\n"
-        "           \"*\" - we do not relay transactions to this peer (relaytxes is false)\n"
+        "           \".\" - we do not relay transactions to this peer (relaytxes is false)\n"
+        "           \"*\" - we do not relay transactions to this peer (relaytxes not available as peer connection is still being set up)\n"
```

Comment:
See https://github.com/bitcoin/bitcoin/pull/26457#discussion_r1024158792, I think the third state (N/A) would not be used during connection startup as intended. I confirmed this by adding a sleep at the beginning of `::VERSION `processing in `ProcessMessage` and querying getpeerinfo with this branch - I'd get `relaytxes= false` but the field is included in the response.